js/ui/calendar/ui.calendar.js

```diff
@@ -169,6 +169,13 @@ const Calendar = Editor.inherit({
              */
             disabledDates: null,
 
+            /**
+             * @name dxCalendarOptions.navigateToSameDate
+             * @type boolean
+             * @default true
+             */
+            navigateToSameDate: true,
```

Comment:
@AlexJameson 
Can you review option name?

Reply:
@DokaRus 
I authored this name ;)

Reply:
But when I had second thought, I've shown this to Roman, We agreed on **focusSameDate**